cartographer_ros/src/submaps_display.cc

```diff
@@ -136,7 +138,7 @@ void SubmapsDisplay::onInitialize() {
       kSubmapTexturesGroup);
 
   scene_manager_->addListener(&scene_manager_listener_);
-  UpdateTopic();
+  UpdateSubmapQueryServiceName();
```

Comment:
Please add a todo to combine this with UpdateTopic(). It's unlikely we ever want to do one without doing the other as well.